core/src/test/scala/io/finch/BodySpec.scala

```diff
@@ -36,29 +36,29 @@ class BodySpec extends FinchSpec {
   it should "not match on streaming requests" in {
     val req = Request()
     req.setChunked(true)
-    body[Foo, Text.Plain].apply(Input.fromRequest(req)).awaitValueUnsafe() shouldBe None
+    body[Foo, Text.Plain].apply(Input.fromRequest(req)).awaitValueUnsafe(dispatcherIO) shouldBe None
```

Comment:
Maybe CE already provides some kind of test base class that instantiates a dispatcher on before and shuts it down on after?

Reply:
Thinking out loud: what if we give up providing this awaitX method on EndpointResult and maybe move that responsibility to users? If you think about it, it's not too bad of an experience after all, given that F is already defined as IO. Folks can just can runSyncUnsafe  (or whatever it's called) on their IO instances in tests directly. We could still make this experience nicer if we provide convenient functions transforming the output, but running it should be something we leave our users to deal with.

Reply:
That's surely an option - but we would have to change our own tests quite a bit. I would leave that for another PR.

Reply:
Yea I like your plan.